bin/resin-dev

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+
```

Comment:
Can we put a loud comment in here documenting that this is for dev use _only_? Might be nice to mention it in the dev section of the readme too.